DependencyInjection/Configuration.php

```diff
@@ -98,6 +98,9 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->scalarNode('remove_token_from_body')->defaultTrue()->end()
                     ->end()
                 ->end()
+                ->scalarNode('logout_firewall')
+                    ->defaultValue('api')
+                    ->info('Name of the firewall that triggers the logout event to hook into (default: api)')
```

Comment:
```suggestion
                    ->info('Name of the firewall that triggers the logout event to hook into (default: api)')
                ->end()
```

Looks like an `end()` call is missing based on the indents.

Reply:
Thanks, fixed.